packages/components/src/components/FindingDetailsPanel.vue

```diff
@@ -0,0 +1,274 @@
+      </div>
+      <div class="section-content">
+        <ul class="overview">
+          <li>Impact Category: {{ impactDomain }}</li>
+          <li v-for="(link, reference) in references" :key="reference">
+            Reference: <a :href="link">{{ reference }}</a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="message">
+      <div class="section-header">
+        <h3>Message:</h3>
+      </div>
+      <p class="code">{{ message }}</p>
+    </section>
+    <section class="stack-trace">
+      <div class="section-header">
+        <h3>Stack Trace</h3>
+      </div>
+      <ul>
+        <li v-for="location in stackLocations" :key="location.uri.path" data-cy="stack-trace">
+          <v-popper
+            class="hover-text-popper"
+            data-cy="popper"
+            :text="location.uri.path"
+            placement="top"
+            text-align="left"
+          >
+            <a href="#" @click.prevent="openInSourceCode(location)">{{
+              displayLocation(location)
+            }}</a>
+          </v-popper>
+        </li>
+      </ul>
+    </section>
+  </div>
+</template>
+
+<script>
+import VPopper from '@/components/Popper.vue';
+
+export default {
+  name: 'v-finding-details-panel',
+  components: {
+    VPopper,
+  },
+
+  props: {
+    finding: {
+      type: Object,
+      required: true,
+    },
+  },
+
+  methods: {
+    displayLocation(location) {
+      const lineNumber = location.range[0] && location.range[0].line + 1;
+      const { truncatedPath } = location;
+      if (lineNumber && lineNumber > 1) {
+        return `${truncatedPath}:${lineNumber}`;
+      }
+      return `${truncatedPath}`;
+    },
+
+    openInSourceCode(location) {
+      let locationStr = location.uri.path;
+      const lineNumber = location.range[0] && location.range[0].line + 1;
+
+      if (lineNumber && lineNumber !== 0) {
+        locationStr += `:${lineNumber}`;
+      }
+
+      this.$root.$emit('viewSource', {
+        location: locationStr,
+      });
+    },
+  },
+
+  computed: {
+    rule() {
+      return this.finding.rule;
+    },
+
+    description() {
+      return this.rule.description;
+    },
+
+    title() {
+      return this.rule.title;
+    },
+
+    impactDomain() {
+      return this.rule.impactDomain;
+    },
+
+    references() {
+      return this.rule.references;
+    },
+
+    message() {
+      return this.finding.finding.message;
+    },
+
+    stackLocations() {
+      return this.finding.stackLocations;
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+.finding-details {
+  display: flex;
+  flex-direction: column;
+  gap: 0;
+  margin-bottom: 2rem;
+  font-size: 0.9rem;
+  border: 1px solid $gray2;
+  border-radius: 0.5rem;
+  margin: 1rem 0;
+
+  section {
+    padding: 0 1rem;
+    .code {
+      color: $white;
+      margin-top: 0;
+      font-family: monospace;
+    }
+  }
+  .heading,
+  section {
+    border-bottom: 3px solid $gray2;
```

Comment:
There's some inconsistency here with the border width. The outer border is `1px` while the inner is `3px`. We're also not adding an outer border anywhere else, but I think this is more of an issue of the finding details needing to be its own panel.
![image](https://user-images.githubusercontent.com/8737782/209199708-86727f05-7e66-4fd3-8a31-a28038a8d0c7.png)

Reply:
The margins look like they could be more consistent here as well